packages/core/src/http.listener.ts

```diff
@@ -16,7 +16,7 @@ export const httpListener = ({ middlewares = [], effects, errorMiddleware }: Htt
   const request$ = new Subject<Http>();
   const effect$ = request$
     .pipe(
-      switchMap(({ req, res }) =>
+      flatMap(({ req, res }) =>
```

Comment:
The `switchMap` operator wont break the http behavior because the `request$` stream will contain only processed requests by node http server (putted via callback), but if we would like to be 1000% sure that it will work we can use `flatMap/mergeMap`.

Instead of `flatMap` I would suggest to use `mergeMap` because the first one is just an alias for the second.

@nlko were you able to reproduce this issue, or you are only guessing?

BTW. thx for contribution!